pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1809,6 +1809,7 @@ func TestRunBindPlugins(t *testing.T) {
 			wantStatus: v1alpha1.Error,
 		},
 	}
+
```

Comment:
revert

Reply:
This seems to be done by the VS Code go plugin, let me revert this